nav2_bt_navigator/behavior_trees/navigate_w_recovery_and_constant_replanning_and_if_path_becomes_invalid.xml

```diff
@@ -0,0 +1,46 @@
+<!--
+  This Behavior Tree replans the global path only if the path becomes invalid and it also has
+  recovery actions specific to planning / control as well as general system issues.
+  This will be continuous if a kinematically valid planner is selected.
+-->
+<root main_tree_to_execute="MainTree">
+  <BehaviorTree ID="MainTree">
+    <RecoveryNode number_of_retries="6" name="NavigateRecovery">
+      <PipelineSequence name="NavigateWithReplanning">
+        <RateController hz="2.0">
+          <RecoveryNode number_of_retries="1" name="ComputePathToPose">
+            <Fallback>
+              <ReactiveSequence>
+                <Inverter>
+                  <GlobalTimeExpired seconds="15"/>
```

Comment:
You're also missing the `path="{path}"` port -- I don't think this works without that